src/components/RightSidebar/Participants/Participant.vue

```diff
@@ -1176,110 +1124,84 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.selected {
-	background-color: var(--color-primary-element-light);
-	border-radius: 5px;
-}
+.participant {
+	// Overwrite NcListItem styles
+	:deep(.list-item) {
+		overflow: hidden;
+		outline-offset: -2px;
+		cursor: default;
 
-.participant-row {
-	display: flex;
-	align-items: center;
-	cursor: default;
-	margin: 4px 0;
-	border-radius: var(--border-radius-pill);
-	height: 56px;
-	padding: 0 4px;
+		a,
+		a * {
+			cursor: default;
+		}
+	}
+
+	&--offline &__user-name {
+		color: var(--color-text-maxcontrast);
+	}
 
-	&.isSearched {
+	&--searched :deep(.list-item) {
 		cursor: pointer;
 	}
```

Comment:
It is not applied and participant is now selectable in list, is it expected ?
![participants](https://github.com/user-attachments/assets/2e877efa-3fb8-4b1f-9e8d-427eaca060d3)

Reply:
styles fixed.
It's expected as it is interactive element `<a/>`. we have it before anyway with `tabindex="0"`, just without visual feedback